lib/services/suite/suite.go

```diff
@@ -1755,6 +1758,7 @@ func (s *ServicesTestSuite) Events(c *check.C) {
 
 // EventsClusterConfig tests cluster config resource events
 func (s *ServicesTestSuite) EventsClusterConfig(c *check.C) {
+	ctx := context.Background()
```

Comment:
Delete this `ctx` definition and name the `context.Context` parameter of the `crud` function below instead.